app/src/main/java/ca/bc/gov/bchealth/ui/travelpass/FetchFederalTravelPassFragment.kt

```diff
@@ -194,7 +194,7 @@ class FetchFederalTravelPassFragment : BaseFragment(R.layout.fragment_fetch_trav
             viewModel.fetchVaccineRecord(
                 phn,
                 patientDataDto.patient.dateOfBirth.toDate(yyyy_MM_dd),
-                patientDataDto.vaccineWithDoses?.doses!!.last()?.date.toDate(yyyy_MM_dd)
+                patientDataDto.vaccineWithDoses?.doses!!.last().date.toDate(yyyy_MM_dd)
```

Comment:
I would rather recommend doing an doses?.let{  } unwrapping operation rather than using !! operator. The .let inside the method, of course.